src/components/timeline/Selection.js

```diff
@@ -385,6 +386,7 @@ export default explicitConnect<OwnProps, StateProps, DispatchProps>({
     previewSelection: getPreviewSelection(state),
     committedRange: getCommittedRange(state),
     zeroAt: getZeroAt(state),
+    mouseTimePosition: getMouseTimePosition(state),
```

Comment:
I would hold off on using this in a component until a follow-up PR where you are actually using it, otherwise it could be confusing for other code readers.